Makefile

```diff
@@ -16,7 +16,11 @@ all: hubble
 hubble:
 	$(GO) build $(if $(GO_TAGS),-tags $(GO_TAGS)) -ldflags "-w -s -X 'github.com/cilium/hubble/pkg.GitBranch=${GIT_BRANCH}' -X 'github.com/cilium/hubble/pkg.GitHash=$(GIT_HASH)' -X 'github.com/cilium/hubble/pkg.Version=${VERSION}'" -o $(TARGET)
 
-release: clean
+release:
+	docker run --rm --workdir /hubble --volume `pwd`:/hubble docker.io/library/golang:1.14.4-alpine3.12 \
+		sh -c "apk add --no-cache make && make local-release"
```

Comment:
In case anyone ever finds this PR and is confused what has stumped us for so long, it was the original usage of `&&`. Instead of running that inside docker, it was running locally and only worked for folks that had `sha256sum` installed locally (completely by-passing docker image...)